src/hera/workflows/models/io/k8s/api/core/v1.py

```diff
@@ -1526,7 +1526,7 @@ class HTTPGetAction(BaseModel):
         description=("Custom headers to set in the request. HTTP allows repeated headers."),
     )
     path: Optional[str] = Field(default=None, description="Path to access on the HTTP server.")
-    port: intstr.IntOrString = Field(
+    port: int = Field(
         ...,
         description=(
             "Name or number of the port to access on the container. Number must be in"
```

Comment:
Interesting that the description is "name or number" 🤔

Reply:
Yea.. and all other ports I looked at, on other objects, are integers. Perhaps it's a backwards compatibility thing that upstream tried to address. I lack the context